ui/pom.xml

```diff
@@ -89,4 +89,4 @@
       <type>xar</type>
     </dependency>
   </dependencies>
-</project>
\ No newline at end of file
+</project>
```

Comment:
change not needed here I think.

Reply:
yes because I used github to edit pom.xml